src/main/java/bio/terra/service/auth/iam/IamResourceType.java

```diff
@@ -6,22 +6,30 @@
 import org.apache.commons.lang3.StringUtils;
 
 public enum IamResourceType {
-  DATAREPO("datarepo"),
-  DATASET("dataset"),
-  DATASNAPSHOT("datasnapshot"),
-  SPEND_PROFILE("spend-profile"),
-  WORKSPACE("workspace");
+  DATAREPO("datarepo", IamResourceTypeEnum.DATAREPO),
```

Comment:
just curious, what does this resource type represent?

Reply:
Yes it's used for global/admin permissions, since some operations are limited to admin users but aren't associated with a specific resource ID.